src/olympia/blocklist/admin.py

```diff
@@ -573,6 +590,8 @@ def changeform_view(self, request, obj_id=None, form_url='',
                 return HttpResponseTemporaryRedirect(
                     reverse('admin:blocklist_blocksubmission_add'))
 
+        extra_context = extra_context or {}
```

Comment:
I'm not really sure what the `extra_context` does or where it comes from, but instead of this could you just default `extra_context` to `{}` instead of `None` on line 582?

Reply:
> I'm not really sure what the extra_context does or where it comes from, 

It's extra data that will be passed to the templates when rendering (the "context" is the dict containing all the dynamic content needed to render).

> but instead of this could you just default extra_context to {} instead of None on line 582?  

In python the default args are reused for the function so if you define a mutable object and modify it then it will still be there _next_ time that function is called.  Currently in this function we're always modifying the object in the same way and only referencing it afterwards but imo it's good practise to take that common gotcha into account (if any of the code above was changed to reference `extra_context` it'd lead to a confusing bug).